src/cpu/aarch64/injectors/jit_uni_binary_injector.hpp

```diff
@@ -612,10 +614,10 @@ class jit_uni_binary_injector_t {
 
     static constexpr int sizeof_reg64 = 8;
     /*
-     * Instructions from SSE/AVX used to compute binary result like vaddps where
+     * Instructions from SSE/sve used to compute binary result like vaddps where
      * second operand is memory, require mem operand to be 16/32 byte explicitly
      * aligned. (Intel Manual chapter 2.4).
-     * Rule is relaxed from AVX2 (Intel Manual chapter 14.9).
+     * Rule is relaxed from sve2 (Intel Manual chapter 14.9).
```

Comment:
This does not look right (in other places as well):
- I think you meant `sve256` and not `sve2`, because sve2 is a superset of `sve`, while `sve256` is a subset of `sve`
- `Intel Manual chapter 14.9`

I would recommend to avoid changes for the implementations outside of what is added as part of this PR. If we want to fix names everywhere across `Aarch64` then I would suggest a separate PR for that.

 **In this PR** please avoid using `x86` extension names (`avx2`, `avx512`, etc.) in brgemm and keep the rest as is.

Reply:
@igorsafo, thanks for sharing the above feedback. We have updated the changes accordingly, kindly support to check and approve the PR.